owncloudComLibrary/src/main/java/com/owncloud/android/lib/resources/status/GetRemoteCapabilitiesOperation.java

```diff
@@ -181,10 +186,38 @@ public GetRemoteCapabilitiesOperation() {
                                 JSONObject respPublic = respFilesSharing.getJSONObject(NODE_PUBLIC);
                                 capability.setFilesSharingPublicEnabled(CapabilityBooleanType.fromBooleanValue(
                                         respPublic.getBoolean(PROPERTY_ENABLED)));
+
                                 if (respPublic.has(NODE_PASSWORD)) {
+                                    JSONObject respPassword = respPublic.getJSONObject(NODE_PASSWORD);
                                     capability.setFilesSharingPublicPasswordEnforced(
                                             CapabilityBooleanType.fromBooleanValue(
-                                                    respPublic.getJSONObject(NODE_PASSWORD).getBoolean(PROPERTY_ENFORCED)));
+                                                    respPublic.getJSONObject(NODE_PASSWORD).
+                                                            getBoolean(PROPERTY_ENFORCED)
```

Comment:
I see you like this one-line-one-statement/condition. But tastes are different

Reply:
In this case is because if I put `getBoolean(PROPERTY_ENFORCED)` along with the previous the line it surpasses 120 columns.

Reply:
I vote for 150 signs